extension/src/entrypoints/popup/App.tsx

```diff
@@ -286,6 +300,47 @@ function App() {
     }
   }, [folders]);
 
+  async function handleLogin(e: React.FormEvent) {
+    e.preventDefault();
+    setLoginError(null);
+    try {
+      const res = await fetch("http://localhost:3001/login", {
+        method: "POST",
+        headers: { "Content-Type": "application/json",
+          "Authorization": ""
+         },
+        body: JSON.stringify({
+          user: { email: loginEmail, password: loginPassword },
+        }),
+        credentials: "omit"
+      });
+      const response = await res.json();
+      const authHeader = res.headers.get("Authorization");
+      let token;
+      if (authHeader && authHeader.startsWith("Bearer ")) {
+        token = authHeader.split(" ")[1];
+      }
+      if (!token) throw new Error("No token received");
+      try {
+        await browser.storage.local.set({ jwt: token });
+      } catch (e) {
+        console.error("Failed to store JWT in browser storage:", e);
+      }
+      console.log("Stored JWT:", token);
```

Comment:
Probably don't want this console log in the final build, or the one showing the retieved jwt.